app/views/recipes/_form.html.erb

```diff
@@ -24,13 +24,25 @@
     <%= f.text_area :description %>
   </div>
   <div class="field">
-    <%= f.label :cook, 'Cook Time(in min)' %><br>
+    <%= f.label :cook, :value => 'Cook Time(in min)' %><br>
```

Comment:
Could we change all ":value =>" to "value:"? They are equivalent but let's stick to latter version since other files follow the latter rule.